.github/workflows/wheels.yml

```diff
@@ -0,0 +1,34 @@
+name: Wheels
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build wheel ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "ubuntu-20.04", "windows-2019", "macos-11" ]
+    env:
+      CIBW_ARCHS_MACOS: x86_64 arm64 universal2
```

Comment:
why do we need all three of them? If we do universal2, we should not need the other specific ones, no?

Reply:
I guess the univerasal2 version of Python (from python.org) can be run in native arm64 mode or in x86_64 compatibility mode, and depending on that pip should select the appropriate wheel.. But I'm actually not sure what is the priority among those three. If it's "prefer the more specific one", then the universal2 will never get used? I can't know for sure as I don't have an M1 machine to test with

Reply:
it looks like projects such as numpy build and publish all three of them to PyPI, we may as well do the same

Reply:
I might be wrong, but I think that universal wheels are only supported on macOS 11.